README.md

```diff
@@ -22,8 +22,8 @@ Es ist aufwendig für Lerngruppen oder Kurse Dateien miteinander zu teilen, wenn
 
 ## Dokumentation
 
-Hier kann die Dokumentation gefunden werden: [Dokumentation](https://hhu-propra2.github.io/abschlussprojekt-mopse/doc/documentation.html)
+Hier kann die Dokumentation gefunden werden: [Dokumentation](https://hhu-propra2.github.io/abschlussprojekt-mopse/)
 
 Javadoc kann hier aufgerufen werden: [Javadoc](https://hhu-propra2.github.io/abschlussprojekt-mopse/javadoc/)
 
-Die REST API Documentation befindet sich hier: [REST API](https://hhu-propra2.github.io/abschlussprojekt-mopse/api/external-api.html)
+Die REST API Documentation befindet sich hier: [REST API](https://hhu-propra2.github.io/abschlussprojekt-mopse/#section-system-scope-and-context)
```

Comment:
why the anchored link?

Reply:
why not? So it is easier to find.

Reply:
the link should lead to the top

Reply:
The documentation link leads to top.

Reply:
no it leads to the system-scope-and-context section, which is below the table of contents and maybe a preamble